### PR TITLE
EVG-7354: schedule patch PR corrections

### PR DESCRIPTION
**Pull request by tgrander**

### CODE REVIEW

File: graphql/resolvers.go [hunk 1]
```diff
@@ -638,10 +638,12 @@ func (r *mutationResolver) SchedulePatch(ctx context.Context, patchID string, re
...
 	patchUpdateReq.BuildFromGqlInput(reconfigure)
 	version, err := r.sc.FindVersionById(patchID)
-	// FindVersionById does not distinguish between nil version err and db err; therefore must check that err
-	// does not contain nil version err values before sending InternalServerError
-	if !strings.Contains(err.Error(), strconv.Itoa(http.StatusNotFound)) {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error occured fetching patch `%s`: %s", patchID, err.Error()))
+	if err != nil {
+		// FindVersionById does not distinguish between nil version err and db err; therefore must check that err
+		// does not contain nil version err values before sending InternalServerError
+		if !strings.Contains(err.Error(), strconv.Itoa(http.StatusNotFound)) {
```
Comment (john-m-liu) [inline graphql/resolvers.go]: You should also handle the error being not found here, currently you'll pass a null version into the function below if this happens

Comment (tgrander) [reply graphql/resolvers.go]: the flow you describe is intentional. version can be nil.

**Comment by tgrander** [comment]
> There are some unaddressed comments from before. For the one that you couldn't get to work, the following diff should work (rename the extension to .diff to git apply it). The reasoning for that is you're setting global state twice by calling functions that the constructor for the testutil package also calls
> 
> [diff.txt](https://github.com/evergreen-ci/evergreen/files/4355061/diff.txt)

THANK YOU!

File: graphql/integration_atomic_test.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/evergreen-ci/evergreen/model/user"
 	"github.com/evergreen-ci/evergreen/service"
 	"github.com/evergreen-ci/evergreen/testutil"
+	_ "github.com/evergreen-ci/evergreen/testutil"
```
Comment (john-m-liu) [inline graphql/integration_atomic_test.go]: This is already imported above, so you don't need this line

**Comment by tgrander** [comment]
> Re: the change in graphql/tests/taskLogs/results.json, I'm not sure what you mean but it's added by testutil. That's the name of a package - are you talking about a specific utility?

using `ConfigureIntegrationTest` changed the result of the test on its own because it prepended a JIRA link. i mentioned it to @SupaJoon (the author of the original test) and he said it seemed safe to update the result.

**Comment by SupaJoon** [comment]
> > Re: the change in graphql/tests/taskLogs/results.json, I'm not sure what you mean but it's added by testutil. That's the name of a package - are you talking about a specific utility?
> 
> using `ConfigureIntegrationTest` changed the result of the test on its own because it prepended a JIRA link. i mentioned it to @SupaJoon (the author of the original test) and he said it seemed safe to update the result.

Updating the Environment changed the output of this function call: https://github.com/evergreen-ci/evergreen/blob/master/rest/model/event.go#L34 
which we use to generate the Jira link

**Pull request merged**